packages/cbioportal-frontend-commons/src/components/scatterPlot/styles.scss

```diff
@@ -4,7 +4,7 @@
         display: block !important;
         max-width: 600px !important;
         div:last-child {
-            white-space: nowrap;
+            // white-space: nowrap;
```

Comment:
@Nesopie thanks so much for this PR.  I think you're right that a good approach is to allow wrapping inside tooltip, however, I'm pretty sure we must have forbidden it for a reason.   Perhaps see if you can use git blame to find the commit in which the this line was introduced.  We might get lucky and figure out why it was this way in the first place.

Reply:
Yeah, I thought about the other stuff breaking. The tooltips inside the scatter points did not have any effect on them which is why I though it was a good idea to make a PR. I'll look into it further

Reply:
@alisman The whole file was introduced in a single commit - not just the single line. Is it safe to assume that this was not done to fix some bug?

Reply:
@Nesopie i see.    not sure.   i wonder if you can add a property to the component "mutlLine" or "wrapText" which would allow us to do this in a targetted way?

Reply:
I mean add a property to the <DefaultTooltip> component.

Reply:
Sure! I'll try doing that.

Reply:
@alisman It looks like the group selector buttons use the react-bootstrap Popover component instead of the DefaultTooltip component. Adding an inline style whiteSpace: normal in the popover component for the group selector buttons seems to fix it. Can I proceed with this?